### DISCUSSION TITLE
Static Site Generation

### DISCUSSION THREAD

**Discussion by fweth** [standardagents/arrow-js#35] [Q&A] [answered]
Is it possible to use arrow-js for static site generation?  Possibly with Astro or some other meta framework?

**Comment by justin-schroeder** [comment, marked as answer]
Definitely possible, it should run just fine using JSDom in Node (its how tests are written), but no effort has been made yet on doing any kind of integration. Perhaps the lowest hanging fruit would be some kind of Vite based SSR plugin.

The only thing ArrowJS is strictly missing from is re-hydration. A "no js" bundler should be fairly easy though. I'm open to suggestions.
